### DISCUSSION TITLE
import net.fabricmc.fabric.api.tool can not be resolved

### DISCUSSION THREAD

**Discussion by DavidChen37** [FabricMC/fabric-api#2074] [Mod Dev Support] [answered]
just this import not working 

![image](https://user-images.githubusercontent.com/79558010/158373833-6e55afcf-90ed-4c83-a689-0f66781d9ab2.png)

**Comment by warjort** [comment, marked as answer]
https://github.com/FabricMC/fabric/pull/1989

**Reply by DavidChen37** [reply]
is there an alternative to FabricToolTags?

**Reply by warjort** [reply]
The fabric tool stuff was pretty much legacy as of 1.17

Vanilla replaced it with tags like "mineable/pickaxe" or "needs_iron_tool" on the block which your tools check when used.
See the wiki for vanilla block and item tags: https://minecraft.fandom.com/wiki/Tag

See for example how ````net.minecraft.item.PickaxeItem```` passes ````BlockTags.PICKAXE_MINEABLE```` to its parent class MiningToolItem and how that uses the tag.
